api/v1/routes/iterations.rb

```diff
@@ -74,13 +74,13 @@ class Iterations < Core
           data['path'] = segments[2..-1].join("/")
         end
 
-        track = Trackler.tracks[data['language']]
+        track = Trackler.tracks[data['language'].downcase]
```

Comment:
On second thought, maybe I am wrong there. Lot of stuff happening with that hack around. Better not to touch that. 😄